SETUP.md

```diff
@@ -56,11 +56,9 @@ Once `nvm` is successfully installed, restart the terminal and run the following
 
 ```bash
 # restart the terminal after installing nvm
-nvm install 8
-nvm install 10
-nvm alias default 10
+nvm install 12
+nvm alias default 12
```

Comment:
Note I use https://github.com/tj/n over nvm and it's great.

Reply:
Good to know. Are you suggesting we change the doc to reference n?

If so I think that change may deserve it's own ticket. I notice that n needs to be installed via npm which means you need node installed before you can use n, so if we did switch to n in the doc we would need to re-write this doc to cover installing the first version of node manually.